packages/logging-winston/test/index.js

```diff
@@ -85,13 +85,43 @@ describe('logging-winston', function() {
       });
     });
 
+    it('should default to logging.write scope', function() {
```

Comment:
These unrelated blocks weren't changed, only re-sorted to match the order of the logic in the constructor.